web/template/layout/status-tag.gotmpl

```diff
@@ -0,0 +1,9 @@
+{{ define "status-tag" }}
+  <strong class="govuk-tag govuk-tag--{{ statusColour .Status }}">
+    {{ if and (eq .Status "Pending") (.Worked) }}
+      Worked, {{ .Status }}
```

Comment:
it will only ever be "Worked, Pending" so better to be explicit?

Reply:
or you could have done this as 

```
{{ if and (eq .Status "Pending") (.Worked) }}Worked, {{ end }{{ .Status }}
```

Reply:
Took the simpler first option